website/www/site/content/en/contribute/release-guide.md

```diff
@@ -632,37 +632,6 @@ See the source of the script for more details or to run commands manually in cas
 
 ## 8. Prepare documents
 
-### Update and Verify Javadoc
-
-The build with `-PisRelease` creates the combined Javadoc for the release in `sdks/java/javadoc`.
-
-The file `sdks/java/javadoc/build.gradle` contains a list of modules to include and exclude, plus a list of offline URLs that populate links from Beam's Javadoc to the Javadoc for other modules that Beam depends on.
-
-* Confirm that new modules added since the last release have been added to the inclusion list as appropriate.
-
-* Confirm that the excluded package list is up to date.
```

Comment:
Let's add where to look for it.

Reply:
These changes are all deleting instructions that are now part of build_release_candidate.sh. What are you looking for here?